services/web/server/src/simcore_service_webserver/projects/_wallets_handlers.py

```diff
@@ -16,7 +16,7 @@
 from .._meta import API_VTAG
 from ..login.decorators import login_required
 from ..security.decorators import permission_required
-from ..wallets import _api as wallet_api
+from ..wallets import api as wallet_api
```

Comment:
TIP: if you use another plugin in here, I would make sure that setup is called in this plugin's setup . i.e in the `project.plugin.py:setup_project` you should do a call to `setup_wallets`. NOTE that the plugin's setup functions are guaranteed to be called only once.

Reply:
I have added routes to the projects.plugin setup_projects